### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Piotr-Wandas**

### CODE REVIEW

File: src/getCoinCombination.test.js [hunk 1]
```diff
@@ -7,4 +7,44 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it('should return [1, 0, 0, 0] for 1 cent',() => {
```
Comment (anna-lymorenko) [inline src/getCoinCombination.test.js:11]: it('returns correct combination for 1 cent',() => {

File: src/getCoinCombination.test.js [hunk 2]
```diff
@@ -7,4 +7,44 @@ describe('getCoinCombination', () => {
...
+
+  it('should return [1, 0, 1, 1] for 36 cent',() => {
+    expect(getCoinCombination(29)).toEqual([1, 0, 1, 1]);
+  });
+
+  it('should return [0, 0, 2, 2] for 70 cent',() => {
+    expect(getCoinCombination(90)).toEqual([0, 0, 2, 2]);
+  });
+
+  it('should return [2, 0, 1, 3] for 87 cent',() => {
```
Comment (anna-lymorenko) [inline src/getCoinCombination.test.js:43]: The test case description ('should return [3, 1, 1, 3] for 93 cent') matches the expected output [3, 1, 1, 3] for the input 93 cents.
If you intended to have a test for 87 cents instead, here's how the test case should be structured:
```
it('returns correct combination for 87 cent', () => {
  expect(getCoinCombination(87)).toEqual([2, 0, 1, 3]);
});

fix it in the rest of tests
```

**Review by anna-lymorenko — CHANGES_REQUESTED** [review]
Double-check the expected outputs to match the correct calculations for each input

File: src/getCoinCombination.test.js (hunk 1 above)
Comment (Piotr-Wandas) [reply src/getCoinCombination.test.js]: Done

File: src/getCoinCombination.test.js (hunk 2 above)
Comment (Piotr-Wandas) [reply src/getCoinCombination.test.js]: Done

**Pull request closed without merging**